.github/workflows/e2e-parallel-full.yml

```diff
@@ -115,6 +115,7 @@ jobs:
           terraform apply -target=module.eks_blueprints -no-color -input=false -auto-approve
           terraform apply -target=module.eks -no-color -input=false -auto-approve
           terraform apply -target=module.eks_blueprints_kubernetes_addons -no-color -input=false -auto-approve
+          terraform apply -target=module.eks_blueprints_addons -no-color -input=false -auto-approve
```

Comment:
Same here

Reply:
This makes it more correct. Because of target apply like we suggest now